packages/vc-handler-ld-local/package.json

```diff
@@ -21,17 +20,21 @@
     "@digitalcredentials/ed25519-verification-key-2020": "^3.2.2",
     "@digitalcredentials/jsonld": "^5.2.1",
     "@digitalcredentials/jsonld-signatures": "^9.3.1",
+    "jsonld-signatures": "5.0.1",
```

Comment:
Hzzz, can't we make it work with the above @Digtalcredentials version? This import will for sure fail with React Native. That is the whole reason we use the digital credentials forks

Reply:
It will fail with React Native for sure: https://github.com/mattrglobal/jsonld-signatures-bbs/issues/177. There are major differences between this version and @digitalcredentials version. I tried to make it work and I got a VC with a non compliant proof and could not sign a VP at all. By the way, there is no version @digitalcredentials/jsonld-signature 5.0.1